backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -117,38 +118,22 @@ export class UserResolver {
     return 'success'
   }
 
-  @Mutation(() => String)
+  @Mutation(() => Boolean)
   async createUser(
     @Args() { email, firstName, lastName, password, language, publisherId }: CreateUserArgs,
-  ): Promise<string> {
-    const payload = {
-      email,
-      first_name: firstName,
-      last_name: lastName,
-      password,
-      emailType: 2,
-      login_after_register: true,
-      language: language,
-      publisher_id: publisherId,
-    }
-    const result = await apiPost(CONFIG.LOGIN_API_URL + 'createUser', payload)
-    if (!result.success) {
-      throw new Error(result.data)
-    }
-
-    const user = new User(result.data.user)
+  ): Promise<boolean> {
     const dbuser = new DbUser()
-    dbuser.pubkey = Buffer.from(fromHex(user.pubkey))
-    dbuser.email = user.email
-    dbuser.firstName = user.firstName
-    dbuser.lastName = user.lastName
-    dbuser.username = user.username
+    dbuser.pubkey = Buffer.from(fromHex('result.data.user.pubkey'))
+    dbuser.email = email
+    dbuser.firstName = firstName
+    dbuser.lastName = lastName
+    dbuser.username = 'username'
```

Comment:
We also must write into the login server database. If any service (like the old frontend) depends on this we should have it there.

Please also look what the login server does to validate the username etc.
What is the process?